ee/tabby-ui/app/(dashboard)/cluster/components/cluster.tsx

```diff
@@ -112,6 +136,105 @@ export default function Workers() {
           cpuInfo={healthInfo.cpu_info}
         />
       </div>
+
+      <Separator className="mt-5" />
+
+      <div>
+        <p className="font-bold">
+          Storage Usage
+        </p>
+        <p className="text-sm text-muted-foreground">Disk space used by Tabby</p>
```

Comment:
```suggestion
        <p className="text-sm text-muted-foreground">Disk Space Utilization by Type</p>
```

Prompt:

<img width="1403" alt="image" src="https://github.com/TabbyML/tabby/assets/388154/fba9210f-cc4f-4ebf-9ebd-0b8ede1c595a">